src/Evolution/DiscontinuousGalerkin/Initialization/QuadratureTag.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Tags/OptionsGroup.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace evolution::dg {
+namespace OptionTags {
+/// The quadrature points to use.
+struct Quadrature {
+  using type = Spectral::Quadrature;
+  using group = ::dg::OptionTags::DiscontinuousGalerkinGroup;
+  static constexpr Options::String help =
+      "The point distribution/quadrature rule used.";
+};
+}  // namespace OptionTags
+
+namespace Tags {
+/// The quadrature points to use initially.
+///
+/// While they could be change during the evolution, it is unclear there is any
+/// reason to do so or that changing them during an evolution would even be
+/// stable.
```

Comment:
If it seems unlikely that we're going to change them, then should we just drop the `Initial` part of the tag name to avoid the mild implication that they will change?

Reply:
I marked them as `Initial` to be consistent with the other initialization actions and because they are removed after initialization. But I'm not attached to that, just wanted to give you the reasoning and see what you think :)